OsmAnd/src/net/osmand/plus/altimeter/QNH.java

```diff
@@ -0,0 +1,28 @@
+package net.osmand.plus.altimeter;
+import java.lang.Math;
+
+
+//http://en.wikipedia.org/wiki/Atmospheric_pressure
+
+class QNH{
+	public static double standardAtmospherePressure = 1013.25; //standard sea level pressure (hPa)
```

Comment:
If these are constants, it should be safer as "final".
(especially the public one)